### ISSUE TITLE
Extention breaks VSCode's Extention Host when importing invalid files in SASS files

### ISSUE THREAD

**Issue by sarpik** [sergiirocks/vscode-ext-color-highlight#53]
Reproduction:

Enable the color highlight extension
Create a new .scss file
Type in exactly this:
@import " ";
(Enough to break the extention host)
I get this behaviour: https://i.imgur.com/V0mxMel.png

You could also type in
@import "./someActuallyValidPath";
then save, then delete the '/' from the import path to make it invalid, and the extention host breaks again.

Anything else I could help out with?

**Comment by fr3fou** [comment]
So I'm having the exact same issue, any news on this?

**Comment by sarpik** [comment]
> So I'm having the exact same issue, any news on this?

Nope, nothing new. This repo seems stale - last commit was October 29..

**Comment by FelesNoctis** [comment]
It's not just invalid imports, it's anything Live SASS Compiler BELIEVES to be invalid. From what I can tell this seems to be when it and/or another extension are unaware of the environment configuration in which the SASS/SCSS is contained. For instance, the project I was working with functions perfectly fine and appears as expected in the program itself, but Live SASS Compiler throws a fit over it.

Regardless, it's not specifically an interaction between these two extensions as far as I can tell, it's an environment fault that may even be a problem with VSCode itself.

**Comment by FelesNoctis** [comment]
https://github.com/ritwickdey/vscode-live-sass-compiler/issues/112
